src/network_models/pm_translator.jl

```diff
@@ -416,20 +424,15 @@ function get_buses_to_pm(buses::IS.FlattenIteratorWrapper{PSY.Bus})
     PM_buses = Dict{String, Any}()
     PMmap_buses = Dict{Int, PSY.Bus}()
 
-    pm_bustypes = Dict{PSY.BusTypes, Int}(
-        PSY.BusTypes.ISOLATED => 4,
-        PSY.BusTypes.PQ => 1,
-        PSY.BusTypes.PV => 2,
-        PSY.BusTypes.REF => 3,
-        PSY.BusTypes.SLACK => 3,
-    )
-
     for bus in buses
+        if PSY.get_bustype(bus) == PSY.BusTypes.ISOLATED
+            continue
```

Comment:
Any idea why this line isn't being covered by the tests? Seems like test_model_decision should cover it.